extern/gallery/en-US/GalleryContents.xml

```diff
@@ -2,29 +2,34 @@
 <GalleryContents xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 	<GalleryUiContents>
 	<GalleryContent>
-		<Header>Colors in Geometry</Header>
-		<Body>You can now apply colors to the geometry that you create and visualize all your analysis data using color coded elements within Dynamo. To access look out for Core > Display in library view. The canvas also comes with a better grid and the ability to visualize vectors.</Body>
-		<ImagePath>colored_geometry.png</ImagePath>
+		<Header>Manage Node and Package Paths</Header>
+		<Body>Use a common shared network folder within an office environment for easy collaboration.</Body>
+		<ImagePath>ManagePaths.png</ImagePath>
 	</GalleryContent>
 	<GalleryContent>
-		<Header>Grouping and Annotation</Header>
-		<Body>You can now select multiple nodes and group them together. The shortcut for grouping is Ctrl + G. For switching between background preview and node view, use Ctrl + B.</Body>
-		<ImagePath>grouping_annotation.png</ImagePath>
+		<Header>Save Design Options with Presets</Header>
+		<Body>Presets allow you to keep track of different states of your graph by capturing the values of its input nodes.</Body>
+		<ImagePath>Presets.png</ImagePath>
 	</GalleryContent>
 	<GalleryContent>
-		<Header>Crash Recovery</Header>
-		<Body>All your working files are now safe even when Dynamo crashes. Look out for recovered files in the start page when you restart Dynamo after a crash.</Body>
-		<ImagePath>crash_recovery.png</ImagePath>
+		<Header>Convert Nodes to Code</Header>
+		<Body>Select groups of nodes and automatically generate the corresponding DesignScript code.</Body>
+		<ImagePath>Node2Code.png</ImagePath>
 	</GalleryContent>
 	<GalleryContent>
-		<Header>Control Multiple Node States</Header>
-		<Body>You can now select multiple nodes at once to toggle on/off the geometry preview.</Body>
-		<ImagePath>multiple_previews.png</ImagePath>
+		<Header>Improved In-Canvas Search and Tooltips</Header>
+		<Body>Search algorithms and keyboard navigation have been improved so that you can quickly search for nodes without using the Library.&#10;Right-click (or Shift + Double-click) over the canvas to bring up search. Hovering over search results reveals helpful tooltips.</Body>
+		<ImagePath>In-CanvasSearch.png</ImagePath>
 	</GalleryContent>
 	<GalleryContent>
-		<Header>Draggable Library Items</Header>
-		<Body>You can now drag items from the library. It helps you place nodes precisely on the first attempt.</Body>
-		<ImagePath>library_draggable.png</ImagePath>
+		<Header>Color on Surfaces</Header>
+		<Body>Smooth color shading can now be applied to surfaces with the Display.BySurfaceColors node.</Body>
```

Comment:
We should not use the term "Smooth color shading". In 3D, smooth shading is shading that incorporates the surface normal into its coloration calculations. This is in contrast to flat shading which only considers the triangle normal.

Suggest:
"Square arrays of color can now be applied..."